pkg/distributor/ingestion_rate_strategy.go

```diff
@@ -72,3 +72,49 @@ func (s *infiniteStrategy) Burst(tenantID string) int {
 	// Burst is ignored when limit = rate.Inf
 	return 0
 }
+
+type localStrategyNativeHistograms struct {
+	limits *validation.Overrides
+}
+
+func newLocalNativeHistogramsIngestionRateStrategy(limits *validation.Overrides) limiter.RateLimiterStrategy {
```

Comment:
Is there any diff between this and the `localStrategy`? I cant see any. If so, should we reuse the same obj? Maybe we can change that to `defaultLocalStrategy`?



Same for globalStrategyNativeHistograms

Reply:
The limit and Burst method of localStrategyNativeHistograms return NativeHistogramsIngestionRate, as opposed to  IngestionRate for localStrategy. 
https://github.com/cortexproject/cortex/blob/0b4cb0d020e1868d1f8eabb040044ae272ae27a3/pkg/distributor/ingestion_rate_strategy.go#L87